app.py

```diff
@@ -2,7 +2,7 @@
 from bmc.models import User, Practice
 
 if (__name__ == '__main__'):
-    bmc_app.run(debug=False)
+    bmc_app.run(debug=True)
```

Comment:
I think the `debug=True` should be removed. In order to debug, we can set the environment variable: in your powershell/cmd, do: `set FLASK_ENV=development`. See it [here](https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features)